src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
`getStatistic` should have 3 lines - 3 private method calls one by one
![image](https://user-images.githubusercontent.com/20326858/237031441-3904dbe5-5c87-43bd-8252-0eeadf15fd1e.png)